docs/usage/runtime.md

```diff
@@ -40,6 +40,14 @@ require("babel").transform("code", { optional: ["runtime"] });
 $ babel --optional runtime script.js
 ```
 
+In Webpack:
+
+```js
+loaders: [
+  { test: /\.js$/, loaders: ['babel?optional=runtime']}
+]
+```
+
```

Comment:
Yeah, I did think that after sending the PR. It was useful to me but after looking at it I thought "actually that likes messy".

I'll remove it.